kinetic/distribution.yaml

```diff
@@ -2328,6 +2328,30 @@ repositories:
       url: https://github.com/ct2034/dockeROS.git
       version: master
     status: developed
+  doosan-robot:
+    doc:
+      type: git
+      url: https://github.com/doosan-robotics/doosan-robot.git
+      version: master
+    release:
+      packages:
+      - doosan_robotics
+      - dsr_bringup
+      - dsr_control
+      - dsr_description
+      - dsr_example_cpp
+      - dsr_example_py
+      - dsr_gazebo
+      - moveit_config_m0609
+      - moveit_config_m0617
+      - moveit_config_m1013
+      - moveit_config_m1509
+      - robotiq_description
```

Comment:
Is this correct?

Reply:
It does look to be a package in the repository, but since it's a fork it should have a name related to the scope of this package such as dsr_robotiq_description